### ISSUE TITLE
Skyhook missing some epidoses runtime

### ISSUE THREAD

**Issue by Phil57** [Sonarr/Sonarr#6209]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Hello

I seems that Skyhook is missing runtime for some episodes on some show, but not all.
These are present on TVDB and for a while, so I don't think it's related to API cache.

Examples:
https://thetvdb.com/series/richard-hammonds-workshop
https://thetvdb.com/series/ice-airport-alaska
https://thetvdb.com/series/japanology-plus

<img width="992" alt="Screenshot 2023-11-24 120046" src="https://github.com/Sonarr/Sonarr/assets/10242493/01a73b85-1aaf-4844-b933-dcee3e15ca45">

### Expected Behavior

Have all runtime that are reported on TVDB

### Environment

```markdown
- OS: Ubuntu
- Sonarr: 4.0.0.731
- Docker Install: Yes 
- Using Reverse Proxy: No
- Browser: Chrome
- Database:
```

### What branch are you running?

Develop

### Trace Logs?

None

**Comment by markus101** [comment]
Please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.sonarr.tv/), or [IRC](https://web.libera.chat/?channels=#sonarr) for metadata issues.

This is not a bug in Sonarr or Skyhook, but looks to be a bug on TheTVDB's side. I've reported the issue to them: https://github.com/thetvdb/v4-api/issues/313
